internal/zinc-compile-core/src/main/scala/sbt/internal/inc/LoggerReporter.scala

```diff
@@ -100,7 +100,8 @@ class LoggerReporter(maximumErrors: Int, logger: ManagedLogger, sourcePositionMa
 
   private def inc(sev: Severity) = count.put(sev, count.get(sev) + 1)
 
-  private def display(p: Problem): Unit =
+  // this is used by sbt
```

Comment:
I think we should start to consider sbt as no more special than any other user of Zinc.

But in the future, for now thanks for unblocking me.